src/main/java/core/basesyntax/figures/Printable.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.figures;
+
+public interface Printable {
+    void draw();
+
+    default String getFormatted(double d) {
+        return String.format("%.2f", d);
+    }
```

Comment:
we shouldn't use interfaces as a place for utility methods
interfaces should say what methods object must have